tests/sls-api-file-list.js

```diff
@@ -72,133 +81,260 @@ describe('File List Api', () => {
+						limit: 60,
+						page: 1
+					});
+
+					sandbox.assert.calledOnceWithExactly(S3.getSignedUrl, 'getObject', bucketParams);
+				}
 			},
-			description: 'Should return a body with an array of results, sending a filter',
-			request: {
-				data: { filters: { name: 'test' } }
+			{
+				before: sandbox => {
+					sandbox.stub(BaseModel.prototype, 'get')
+						.resolves([rowGetted, rowGetted]);
+					sandbox.stub(BaseModel.prototype, 'getTotals')
+						.resolves({ total: 2 });
+					sandbox.stub(S3, 'getSignedUrl').resolves(url);
+				},
+				description: 'Should return a body with an array of results, with each row formatted (two image type row)',
+				request: {},
+				session: true,
+				response: {
+					body: [rowFormatted, rowFormatted],
+					headers: {
+						'x-janis-total': 2
+					},
+					code: 200
+				},
+				after: (response, sandbox) => {
+					sandbox.assert.calledOnceWithExactly(BaseModel.prototype.getTotals);
+					sandbox.assert.calledOnceWithExactly(BaseModel.prototype.get, {
+						limit: 60,
+						page: 1
+					});
+
+					sandbox.assert.calledTwice(S3.getSignedUrl);
+					sandbox.assert.calledWithExactly(S3.getSignedUrl, 'getObject', bucketParams);
+				}
 			},
-			session: true,
-			response: {
-				body: [rowFormatted],
-				headers: {
-					'x-janis-total': 1
+			{
+				before: sandbox => {
+					sandbox.stub(BaseModel.prototype, 'get')
+						.resolves([rowGetted]);
+					sandbox.stub(BaseModel.prototype, 'getTotals')
+						.resolves({ total: 1 });
+					sandbox.stub(S3, 'getSignedUrl').rejects();
 				},
-				code: 200
+				description: 'Should return 500 if fail getSignedUrl',
+				request: {},
+				session: true,
+				response: {
+					code: 500
+				},
+				after: (response, sandbox) => {
+					sandbox.assert.calledOnceWithExactly(BaseModel.prototype.getTotals);
+					sandbox.assert.calledOnceWithExactly(BaseModel.prototype.get, {
+						limit: 60,
+						page: 1
+					});
+
+					sandbox.assert.calledOnceWithExactly(S3.getSignedUrl, 'getObject', bucketParams);
+				}
 			},
-			after: (response, sandbox) => {
-				sandbox.assert.calledOnce(BaseModel.prototype.get);
-				sandbox.assert.calledWithExactly(BaseModel.prototype.get, {
-					filters: { name: 'test' },
-					limit: 60,
-					page: 1
-				});
-				sandbox.assert.calledOnce(BaseModel.prototype.getTotals);
-				sandbox.assert.calledWithExactly(BaseModel.prototype.getTotals);
-			}
-		},
-		{
-			description: 'Should return a 400 when send a wrong filter',
-			request: {
-				data: { filters: { wrongFilter: 'Testing' } }
+			{
+				before: sandbox => {
+					sandbox.stub(BaseModel.prototype, 'get')
+						.resolves([rowGetted]);
+					sandbox.stub(BaseModel.prototype, 'getTotals')
+						.resolves({ total: 1 });
+					sandbox.stub(S3, 'getSignedUrl').rejects({
+						statusCode: 404
+					});
+				},
+				description: 'Should return 200 with valid data but file not exist in s3',
```

Comment:
La descripción del error no es clara, debería ser algo mas como "Should return 200 with valid data also when file not exists in S3"